applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -37,60 +37,51 @@ ZBUS_CHAN_DEFINE(le_audio_chan, struct le_audio_msg, NULL, NULL, ZBUS_OBSERVERS_
 #define CIS_CONN_RETRY_TIMES	   5
 #define CIS_CONN_RETRY_DELAY_MS	   500
 
-struct le_audio_headset {
+struct le_audio_unicast_server {
 	char *ch_name;
-	uint8_t num_sink_eps;
-	uint8_t num_source_eps;
-	struct bt_bap_stream sink_stream;
-	bool waiting_for_sink_disc;
-	struct bt_bap_ep *sink_ep;
-	struct bt_audio_codec_cap sink_codec_cap[CONFIG_CODEC_CAP_COUNT_MAX];
-	struct bt_bap_stream source_stream;
-	bool waiting_for_source_disc;
-	struct bt_bap_ep *source_ep;
-	struct bt_audio_codec_cap source_codec_cap[CONFIG_CODEC_CAP_COUNT_MAX];
 	struct bt_conn *headset_conn;
```

Comment:
Since we changed the name from headset to unicast_server, not sure if we should also replace all the name which has headset*

Reply:
Good point, will change it